### PR TITLE
Add LINTLY_FILE_OVERRIDE envvar

### PR DESCRIPTION
**Pull request by scriptsrc**
We can have super-linter set the `LINTLY_FILE_OVERRIDE` envvar to always include full path and filename. We shouldn't need the linting tool (like terrascan) to output the full path for us anymore.

### CODE REVIEW

**Review by nandusekarv10 — COMMENTED** [review]
Do we need to update readme file with `LINTLY_FILE_OVERRIDE` variable usage?

**Review by nandusekarv10 — APPROVED** [review]

**Comment by scriptsrc** [comment]
> Do we need to update readme file with `LINTLY_FILE_OVERRIDE` variable usage?

yeah probably. good catch.

File: lintly/parsers.py [hunk 1]
```diff
@@ -22,6 +22,14 @@ def _normalize_path(self, path):
         """
         Normalizes a file path so that it returns a path relative to the root repo directory.
         """
+        # Override the file/path provided by the tool with what is provided
```
Comment (sarahc23) [inline lintly/parsers.py:25]: To add on to @nandusekarv10 's comment, I'd even go further to say that this entire comment *belongs* in the README (i.e., this comment is optional here).

(Also you have an extra space/indent on the next line.)

**Review by sarahc23 — APPROVED** [review]

File: lintly/parsers.py (hunk 1 above)
Comment (scriptsrc) [reply lintly/parsers.py:25]: Added to readme.

Extra space is intentional :).

Comment (sarahc23) [reply lintly/parsers.py:25]: Then two spaces maybe? :) One space is too tiny to indicate 'this is an indent' to me. (Not a big deal regardless, I'm just nit-picking here.)

**Review by sarahc23 — APPROVED** [review]

File: lintly/parsers.py (hunk 1 above)
Comment (scriptsrc) [reply lintly/parsers.py:25]: added an extra space

**Pull request merged**